integration/integration-end-to-end.spec.ts

```diff
@@ -25,7 +25,7 @@ const EblNumberField = Selector("input#root_blNumber");
 const previousDocumentButton = Selector("[data-testid='previous-document-button']");
 const nextDocumentButton = Selector("[data-testid='next-document-button']");
 const fileNameField = Selector("[data-testid='file-name-input']");
-const goBack = ClientFunction(() => window.history.back());
+const goBack = ClientFunction(() => window.close());
```

Comment:
go back = close ?

Reply:
This function opens a new tab, by right the function we should be using is `close` else test fails.

Reply:
me will rename it to close